src/main/features/claim/routes/amount.ts

```diff
@@ -4,16 +4,17 @@ import { Paths } from 'claim/paths'
 
 import { Form } from 'forms/form'
 import { FormValidator } from 'forms/validation/formValidator'
-import ClaimValidator from 'app/utils/claimValidator'
-import ClaimAmountBreakdown from 'forms/models/claimAmountBreakdown'
+import { ClaimValidator } from 'app/utils/claimValidator'
+import { ClaimAmountBreakdown, MAX_NUMBER_OF_ROWS } from 'claim/form/models/claimAmountBreakdown'
 
 import { ErrorHandling } from 'common/errorHandling'
 import { DraftService } from 'common/draft/draftService'
 
 function renderView (form: Form<ClaimAmountBreakdown>, res: express.Response): void {
   res.render(Paths.amountPage.associatedView, {
     form: form,
-    totalAmount: form.model.totalAmount()
+    totalAmount: form.model.totalAmount(),
+    canAddMoreRows: form.model.rows.length < MAX_NUMBER_OF_ROWS
```

Comment:
A `canAddMoreRows` method could exists in the domain model so we do not have to export `MAX_NUMBER_OF_ROWS`